spec/shopify_transporter/shopify_transporter_spec.rb

```diff
@@ -306,6 +329,94 @@ def tmpfile(content)
     end
   end
 
+  describe ShopifyTransporter::Exporters::Exporter do
+    class SomePlatformExporter
+      def export
+        [{ foo: 'bar' }]
+      end
+    end
+
+    it 'writes exported data to file' do
```

Comment:
Add `describe '#run' do`... as I think all these tests are describing the `run` method.